### PR TITLE
some docs

### PR DESCRIPTION
**Pull request by BASM**

### CODE REVIEW

**Comment by BASM** [comment]
More info about prepare, build and run DF

File: README.md [hunk 1]
```diff
@@ -17,8 +17,66 @@ Implemented:
...
+
+Copy:
+* target/release/libdfint_hook.so to libdfhooks.so
+* [font](https://github.com/dfint/update-data/tree/main/store/fonts) to  ./data/art/curses_640x300.png
+* [encoding](https://github.com/dfint/update-data/tree/main/store/encodings) to dfint-data/encoding.toml
+* [offsets](https://github.com/dfint/update-data/tree/main/store/offsets) to dfint-data/offsets.toml
+* [dictionary](https://github.com/dfint/autobuild/tree/main/translation_build/csv/) to dfint-data/dictionary.csv
+* [config](https://github.com/dfint/update-data/blob/main/store/config.toml) to ./dfint-data/config.toml
+
+#### lanch
```
Comment (insolor) [inline README.md:74]: Launch

File: README.md [hunk 2]
```diff
@@ -17,8 +17,66 @@ Implemented:
...
+
+Run
+
+```shell
+./dwarfort
+```
```
Comment (insolor) [inline README.md:80]: or `Dwarf Fortress.exe`, windows version is supported too

File: README.md [hunk 3]
```diff
@@ -17,8 +17,66 @@ Implemented:
...
+
+Install nightly-2022-11-06 toolchain from rustup:
+
+```shell
+rustup install nightly-2022-11-06-x86_64-unknown-linux-gnu
+
+```
+
+#### build
```
Comment (insolor) [inline README.md:55]: In general, it's better to make the first letter of a header capital, and don't use h4 and smaller headers.

**Review by insolor — CHANGES_REQUESTED** [review]

**Comment by BASM** [comment]
I wrote that the build&launch is the same for Windows. Unfortunately, I am unable to test the instructions on Windows, so I cannot correct them for that operating system. I suspect that the name of the executable file is not the only one difference.

**Comment by insolor** [comment]
@BASM building process on Windows is more or less the same, just the name of the library is dfint_hook.dll instead of libdfint_hook.so. See build workflow: https://github.com/dfint/df-steam-hook-rs/blob/main/.github/workflows/build.yml

**Comment by shevernitskiy** [comment]
That is right, the same cli command, output as dll.

**Review by insolor — APPROVED** [review]

**Pull request merged**